home.php

```diff
@@ -0,0 +1,69 @@
+<html>
+   
+<?php
+ include('config/init.php');
+ echoHeader('Courtyard Collective');
+ 
+ if(isset($_REQUEST['newEmail'])){
```

Comment:
The form is in the footer, so I made it apart of my echoFooter() function.

Reply:
Ok. So that's fine, but the problem is that it currently has `action=''` so it's going to submit to whatever page the user is already on. But you're only handling the form in `home.php`, so on any other page it won't work (I think). There are two ways around this:

1) Put the handling of this form (like, lines 7-11) in init.php (or better yet, put it in a function and call that function at the bottom of init.php) so it runs in every page.

2) Have the form submit to a specific page (e.g. `/subscribeToNewsletter.php`) so that no matter where they submit it from, you only have to handle it on that one page.

Let me know if you want to discuss these options more.